.github/workflows/test-runner-nightly-perf.yml

```diff
@@ -0,0 +1,54 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Sample Nightly Perf Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 14 * * *' # run at 2 PM UTC, 9 AM EST.
```

Comment:
Why 9 AM EST? That's going to overlap with the start of the workday for Cambridge.

Reply:
Team can decide when's an appropriate time to trigger the test. I updated it to 1AM ET.